etest/docker/container.py

```diff
@@ -4,7 +4,6 @@
 # etest is freely distributable under the terms of an MIT-style license.
 # See COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-import re
```

Comment:
I remember taking that out ._. whoops

Reply:
It happens.  I've made the checks requirements more strict so hopefully that will help these things being missed in the future.  Let me know if anything causes issues though.